src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -440,7 +440,11 @@ public bool ResetCaches
         public bool EnableNodeReuse
         {
             get { return _enableNodeReuse; }
-            set { _enableNodeReuse = value; }
+            set {
+#if FEATURE_NODE_REUSE
+                _enableNodeReuse = value;
```

Comment:
What about erroring here instead of silently ignoring the new value? Could be too disruptive though, since the silent behaviour should still lead to a successful build ...